cmd/operator_install.go

```diff
@@ -83,6 +83,9 @@ func operatorInstall(config *operatorInstallCommandConfig) error {
 		if existingOperatorWatchspace == "" {
 			existingOperatorWatchspace = "all namespaces"
 		}
+		if existingOperatorWatchspace != operatorNamespace {
```

Comment:
`getOperatorWatchspace` returns a string, but it may be a CSV list of watchspaces, which needs to be processed. Appsody will always install an operator with a single watchspace, but Kabanero may install operators with multiple watchspaces. The string needs to be processed using `getWatchSpaces` as we do in `operatorExistsWithWatchspace`.